subsciber_topic: "/camera_1"
image_publisher_topic: "canny_image"